src/features/import-maps.js

```diff
@@ -5,7 +5,8 @@ import { baseUrl, resolveAndComposeImportMap, hasDocument, resolveUrl, IMPORT_MA
 import { systemJSPrototype, getOrCreateLoad } from '../system-core.js';
 import { errMsg } from '../err-msg.js';
 
-var importMapPromise = Promise.resolve({ imports: {}, scopes: {}, depcache: {} });
+var importMapPromise = Promise.resolve();
+export var importMap = { imports: {}, scopes: {}, depcache: {}, integrity: {} };
```

Comment:
Could you help me understand this change? Why does support of script integrity require synchronous access to the import map?

Reply:
This is an unrelated refactoring allowing access to the import map uncontextually via `import { importMap }`.